proxy/api/src/http/identity.rs

```diff
@@ -23,6 +24,17 @@ fn create_filter(
         .and_then(handler::create)
 }
 
+/// `PUT /`
```

Comment:
Since this only updates the `Metadata`, I'd prefer to namespace it accordingly.

Reply:
What else could it `PUT` though? :thinking:

Reply:
I get your question but note how all the other endpoints aim at the whole `Identity` type, whereas this only aims at the identity Metadata:

- `POST /` create an `Identity`
- `GET  /<id>` gets an `Identity`
- `GET /` gets a list of `Identity`

Thus, seeing `PUT /` for me would mean "update an `Identity`", not a sub-part of an `Identity`. So I'd rather have an explicit, clear, and namespaced endpoint which would still make sense if, in the future, we introduce some other updatable field in `Identity`. :man_shrugging:

Reply:
Ya, makes sense :+1:

Reply:
This `PUT` can update everything there is to update about an `Identity`. Look at `create`, it too accepts only a `Metadata` to describe a whole new identity. I'm fine with moving it into a separate namespace, but IMO it makes sense to keep it here as `update an Identity`.

Reply:
We can also do that, sure, but then I'd expect the endpoint to operate on `Identity` and not `Metadata`.

Reply:
That's a good point, we're operating on an `Identity` and then suddenly on a `Metadata`, that's messy :thinking:

On the other hand we simply can't have the same JSON for getting an identity and creating/updating it, they transport different sets of parameters. To introduce segregation based on the JSONs, we'd need to put getters from setters in separate namespaces despite them working on the same thing.

I don't have strong feeling about it, I'm just nitpicking now and I believe that your solution will be fine :)

Reply:
Let's leave it as is and change when/if needed